app/util/cmr.ts

```diff
@@ -453,3 +455,16 @@ export async function belongsToGroup(
   const response = await cmrSearchBase(path, null, token, { 'X-Harmony-User': username });
   return response.status === 200 && (response.data as string[]).indexOf(username) !== -1;
 }
+
+/**
+ * Return all non-inherited links with rel ending in /data# when no regex is provided.
+ *
+ * @param granule - The granule to obtain links from
+ * @returns An array of granule links
+ */
+export function filterGranuleLinks(
+  granule: CmrGranule,
+): CmrGranuleLink[] {
+  return granule.links.filter((g) => (g.rel.endsWith('/data#') || g.rel.endsWith('/service#'))
```

Comment:
nit: probably slightly more efficient to reorder the terms in your && clause so the test for inherited links short circuits the other tests if it fails. Also, you could replace the two `endsWtih` checks with a single regex.

Reply:
Moved the inherited to the first check.